.travis.yml

```diff
@@ -1,145 +1,148 @@
 # Created with https://github.com/dart-lang/mono_repo
 language: dart
 
-# Hand-edited
```

Comment:
Need to restore this one... seem familiar? lol

Reply:
Added back.